githubdelivery/src/main/java/io/dockstore/githubdelivery/GithubDeliveryS3Client.java

```diff
@@ -181,15 +181,15 @@ private void submitGitHubDeliveryEventsByKey(String key, WorkflowsApi workflowsA
         try {
             String body = getObject(key);
             JsonObject jsonObject = GSON.fromJson(body, JsonObject.class);
-            if (jsonObject.get("action").getAsString().equals("added") || jsonObject.get("action").getAsString().equals("removed")) {
+            if (jsonObject.get("action") != null) { //Instatllation and uninstallation events have action property
```

Comment:
(Apologies if any of the following was detailed previously...)

I was going to suggest that we check the event type prior to checking type-specific fields, similar to how the lambda processes the events, because otherwise, we're susceptible to bugs if we try to process events with unexpected types, new fields, etc.

But, alas, so far as I can tell, the event type isn't part of the event body (!), it arrives in the HTTP header `X-GitHub-Event`:

https://docs.github.com/en/webhooks/webhook-events-and-payloads#delivery-headers

A check of the logged S3 objects, which contains the body, confirms.  No type in there.

So, maybe we should log [at least some of] the data in the event's HTTP headers, in addition to its body?